src/components/experiment-tracking/run-dataset/run-dataset.js

```diff
@@ -162,25 +194,30 @@ function buildDatasetDataMarkup(
             component="div"
             className="details-dataset__tranistion-group-wrapper"
           >
-            {datasetValues.map((data, index) => (
-              <CSSTransition
-                key={data.runId}
-                timeout={300}
-                classNames="details-dataset__value-animation"
-                enter={isSingleRun ? false : true}
-                exit={isSingleRun ? false : true}
-              >
-                <span
-                  className={classnames('details-dataset__value-header', {
-                    'details-dataset__value-header--comparison-view':
-                      index === 0 && enableComparisonView,
-                  })}
+            {datasetValues.map((data, index) => {
+              return (
+                <CSSTransition
+                  key={data.runId}
+                  timeout={300}
+                  classNames="details-dataset__value-animation"
+                  enter={isSingleRun ? false : true}
+                  exit={isSingleRun ? false : true}
```

Comment:
I don't see anything new in this section or the new code below, so you could revert back to the implicit return if you'd like.